civihr_default_theme/scss/pages/_onboarding-wizard.scss

```diff
@@ -722,3 +723,9 @@ $carousel-box-shadow-color: rgba(#3f3c55,0.3);
     }
   }
 }
+
+// Specific to datepicker
```

Comment:
this is too ad-hoc. please explain why it is needed and why for this specific page only

Reply:
> explain why it is needed?

Actually, when using `enable-select2.js` is included in page all `select` will be applied `select2` which will add extra class `select2-container` which in turn adds `margin: 0 5px;`. Bus since the `enable-select2.js` is not loaded we will never get `select2-container` class and not margin as well. So, the margin is added here to display gap between the fields.

> why for this specific page only?

This partial focuses for the styles in onboarding form page and the margin between fields is for the datepicker within this page only.